cmd/bblfsh/server.go

```diff
@@ -13,24 +13,39 @@ type serverCmd struct {
 	Address     string `long:"address" description:"server address to bind to" default:"0.0.0.0:9432"`
 	RuntimePath string `long:"runtime-path" description:"runtime path" default:"/tmp/bblfsh-runtime"`
 	Transport   string `long:"transport" description:"default transport to fetch driver images (docker, docker-daemon)" default:"docker"`
+	JSON        bool   `long:"json" description:"start a JSON REST server instead of a gRPC server"`
```

Comment:
Can we make it `--rest`? Looks more clear than just `--json`, which is used in other places in Babelfish for other stuff. Other than that, good to og.